build.sbt

```diff
@@ -58,7 +58,10 @@ val commonSettings = commonSmlBuildSettings ++ ossPublishSettings ++ Seq(
     releaseStepCommand("sonatypeBundleRelease"),
     pushChanges
   ),
-  ideSkipProject := (scalaVersion.value == scala2_13),
+  ideSkipProject := (scalaVersion.value == scala2_13)
+)
+
+val commonSettingsJVM: Seq[Def.Setting[_]] = commonSettings ++ Seq(
```

Comment:
I know this is really minor, but these builds are complex, so it would be great to have them as similar as possible (them = tapir & sttp client). Can we rename this to `commonJvmSettings`? :)

Reply:
Done